### ISSUE TITLE
Experimental features feedback notification

### ISSUE THREAD

**Issue by slemeur** [containers/podman-desktop#10228]
### Is your feature request related to a problem? Please describe

We would love more feedback from our users on Experimental features. This task is to in context ask them for their feedback as they use the feature. 

Final design can be found here: https://github.com/podman-desktop/podman-desktop/issues/10224#issuecomment-2636084884 

### Describe the solution you'd like

Final design can be found here: https://github.com/podman-desktop/podman-desktop/issues/10224#issuecomment-2636084884 

This task also includes implementation on the current running experiments:

1. `New task manager`, takes an action on the new task manager, show this dialog (`clear all`, `hide task`)
2. `Docker compatibility`, after the user has enabled the Docker tool compatibility or when the user switches docker context
3. `Statusbar Providers`, when the user pins or unpins a provider from the statusbar
4. `Tasks in the statusbar` after a user clicks on a task on the statusbar, wait 2 minutes, then show dialog

**Comment by gastoner** [comment]
@slemeur @Firewall I have a question: 
When the user clicks, e.g., "notify tomorrow" in the Task Manager, should the feedback for Task Manager be shown tomorrow, e.g., after startup anywhere, let's say, the Kubernetes page or settings page/ Dashboard, or when the user opens the Task Manager again?

**Comment by Firewall** [comment]
@gastoner When the user opens the task manager again. This way they can give feedback about the feature they have just used.

**Comment by gastoner** [comment]
@Firewall Also I think the Docker compatibility is now not experimental anymore, but e.g. kubernetes experimental is

**Comment by Firewall** [comment]
That's true. Ideally we make this a habit for every experimental feature that we add in the future. We will always want that feedback!

**Comment by rujutashinde** [comment]
@gastoner based on today's stand-up discussion moving this to the next release, as soon as we have the release 1.21.0 out we can merge this issue, as we would need some more run time
